docs/integrations/amazon-aws/amazon-opensearch-service.md

```diff
@@ -8,20 +8,345 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+   * Time Zone. Select Ignore time zone from the log file and instead use, and select UTC from the dropdown.  
+   * Timestamp Format. Select Automatically detect the format.  
+   * Enable Multiline Processing. Select the Detect messages spanning multiple lines check box, and select Infer Boundaries.  
+2. Click Save.
+
+### **Field in Field Schema**
+
+1. [Classic UI](https://help.sumologic.com/docs/get-started/sumo-logic-ui-classic/). In the main Sumo Logic menu, select Manage Data \> Logs \> Fields.  
+   [New UI](https://help.sumologic.com/docs/get-started/sumo-logic-ui/). In the top menu select Configuration, and then under Logs select Fields. You can also click the Go To... menu at the top of the screen and select Fields.  
+2. Search for the `"domainname"` field.  
+3. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/docs/manage/fields/#manage-fields).
+
+### **Field Extraction Rule(s)**
+
+Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/docs/manage/field-extractions/create-field-extraction-rule/).
+
+```
+
+Rule Name: AwsObservabilityESCloudTrailLogsFER
+Applied at: Ingest Time
+Scope (Specific Data): account=* eventname eventsource \"es.amazonaws.com\"
+```
+
+Parse Expression:
+
+```
+
+| json "userIdentity", "eventSource", "eventName", "awsRegion", "recipientAccountId", "requestParameters", "responseElements" as userIdentity, event_source, event_name, region, recipient_account_id, requestParameters, responseElements nodrop 
+| where event_source = "es.amazonaws.com" 
+| json field=userIdentity "accountId", "type", "arn", "userName"  as accountid, type, arn, username nodrop 
+| parse field=arn ":assumed-role/*" as user nodrop 
+| parse field=arn "arn:aws:iam::*:*" as accountid, user nodrop 
+| json field=requestParameters "domainName" as domainname 
+| if (isBlank(accountid), recipient_account_id, accountid) as accountid 
+| toLowerCase(domainname) as domainname 
+| "aws/es" as namespace 
+| fields region, namespace, domainname, accountid
+```
+
+## **Centralized AWS CloudTrail Log Collection**
+
+In case, you have a centralized collection of CloudTrail logs and are ingesting them from all accounts into a single Sumo Logic CloudTrail log source, create the following Field Extraction Rule to map a proper AWS account(s) friendly name/alias. Create it if not already present or update it as required.
+
+* Rule Name: AWS Accounts  
+* Applied at: Ingest Time  
+* Scope (Specific Data): `_sourceCategory=aws/observability/cloudtrail/logs`  
+* Parse Expression: Enter a parse expression to create an “account” field that maps to the alias you set for each sub account. For example, if you used the “dev” alias for an AWS account with ID "528560886094" and the “prod” alias for an AWS account with ID "567680881046", your parse expression would look like:
```

Comment:
```suggestion
```

#### Parse Expression

Enter a parse expression to create an “account” field that maps to the alias you set for each sub account. For example, if you used the `“dev”` alias for an AWS account with ID `"528560886094"` and the `“prod”` alias for an AWS account with ID `"567680881046"`, your parse expression would look like:
```